pkg/cmd/roachtest/tests/kv.go

```diff
@@ -170,9 +167,14 @@ func registerKV(r registry.Registry) {
 				sequential = " --sequential"
 			}
 
-			var envFlags string
-			if e := os.Getenv(envKVFlags); e != "" {
```

Comment:
Probably fine to remove but do we know if anyone was using this for flags besides duration?  If so that's more reason to go for a `generic params` approach in the future.

Reply:
Didn't find any usage of these existing flags anywhere in this repo.